lib_standard_app/debug.c

```diff
@@ -19,20 +19,58 @@
 
 #include "os.h"
 #include "io.h"
+#include "debug.h"
 
 #ifdef HAVE_NBGL
 #include "nbgl_use_case.h"
 #endif
 
-#ifdef HAVE_DEBUG_THROWS
-static char errordata[20];
+#if defined(HAVE_LEDGER_ASSERT_DISPLAY) || defined(HAVE_DEBUG_THROWS)
+
+#ifndef ASSERT_BUFFER_LEN
+#define ASSERT_BUFFER_LEN 24
+#endif
+char assert_buffer[ASSERT_BUFFER_LEN];
 
+#ifdef HAVE_DEBUG_THROWS
 WEAK void app_throw_info(unsigned int exception, unsigned int lr_val)
```

Comment:
What do you think of dropping all the `WEAK` here? I don't see a reason for an app to change those behaviors. do you?
This might even solve our double `WEAK` issue in `os.c`?